genesis/engine/entities/rigid_entity/rigid_entity.py

```diff
@@ -1957,10 +1957,53 @@ def get_links_ang(self, links_idx_local=None, envs_idx=None, *, unsafe=False):
         links_idx = self._get_idx(links_idx_local, self.n_links, self._link_start, unsafe=True)
         return self._solver.get_links_ang(links_idx, envs_idx, unsafe=unsafe)
 
+    @gs.assert_built
+    def get_links_accelerometer_data(self, links_idx_local=None, envs_idx=None, *, imu=False, unsafe=False):
```

Comment:
It would be nice to have an example about `get_links_accelerometer_data` and show its difference from `get_links_acc `

Reply:
I would rather merge this first and create a PR for the example next week if you are ok. Easier to manage with all the apes in the pipe  :)